app/src/main/java/edu/uc/coffeens/stocktracker/services/StockService.kt

```diff
@@ -20,7 +21,7 @@ class StockService {
         val call = service?.getAllStocks()
         call?.enqueue(object : Callback<ArrayList<Stock>> {
             override fun onFailure(call: Call<ArrayList<Stock>>, t: Throwable) {
-                print("Unable to retrieve service response.")
+                Log.e("Retrofit", "unable to receive service response")
```

Comment:
Changed from print message to Log.e for error messaging https://developer.android.com/reference/kotlin/android/util/Log

Reply:
Good.